markdown-code-checker.py

```diff
@@ -0,0 +1,90 @@
+import marko as md
+import os
+import argparse
+import sys
+
+def extract_inline_code(file_path, languages):
+    """extract inline code, language from markdown"""
+    
+    with open(file_path, "r") as f:
+        content = f.read()
+
+    parser = md.parser.Parser()
+    ast = parser.parse(content)
+
+    code_snippet_count = 0
+    for child in ast.children:
+        # TODO: add a way to exclude a code snippet
+        if isinstance(child, md.block.FencedCode) and child.lang in languages:
+            code_snippet_count+=1
```

Comment:
In other repos / python tools we run `black` to format the source files. Can you do the same here?